tests/dragonfly/replication_test.py

```diff
@@ -907,36 +911,36 @@ async def test_script_transfer(df_local_factory):
 @pytest.mark.asyncio
 async def test_role_command(df_local_factory, n_keys=20):
     master = df_local_factory.create()
-    replica = df_local_factory.create(logtostdout=True)
+    replica = df_local_factory.create()
 
     df_local_factory.start_all([master, replica])
 
-    c_master = aioredis.Redis(port=master.port)
-    c_replica = aioredis.Redis(port=replica.port)
+    c_master = create_client(master.port)
+    c_replica = create_client(replica.port)
 
-    assert await c_master.execute_command("role") == [b"master", []]
+    assert await c_master.execute_command("role") == ["master", []]
     await c_replica.execute_command(f"REPLICAOF localhost {master.port}")
     await wait_available_async(c_replica)
 
     assert await c_master.execute_command("role") == [
-        b"master",
-        [[b"127.0.0.1", bytes(str(replica.port), "ascii"), b"stable_sync"]],
+        "master",
+        [["127.0.0.1", str(replica.port), "stable_sync"]],
     ]
     assert await c_replica.execute_command("role") == [
-        b"replica",
-        b"localhost",
-        bytes(str(master.port), "ascii"),
-        b"stable_sync",
+        "replica",
+        "localhost",
+        str(master.port),
+        "stable_sync",
     ]
 
     # This tests that we react fast to socket shutdowns and don't hang on
     # things like the ACK or execution fibers.
     master.stop()
     await asyncio.sleep(0.1)
     assert await c_replica.execute_command("role") == [
-        b"replica",
-        b"localhost",
-        bytes(str(master.port), "ascii"),
+        "replica",
+        "localhost",
+        str(master.port),
         b"connecting",
```

Comment:
This is what causing the reg to fail, remove the `b`

Reply:
i have not sent it for review, it was on purpose.